### PR TITLE
File normalizer

### PR DESCRIPTION
**Pull request by bpbond**
Fix line endings and missing final lines in input files, and compress as necessary. Addresses #39

### CODE REVIEW

**Comment by bpbond** [comment]
Damn it, forgot to update the test code.

**Comment by bpbond** [comment]
Ping @rplzzz if you have time.

**Comment by pralitp** [comment]
@bpbond, @rplzzz I was about to start on the process to knock out #865.  One of the things that wasn't clear was:

> energy-data/assumptions/* (hard to say since Robert just changed line endings on all the files here)

I was planning on just re-dump all of those assumption files.  Is that going to conflict with changes here?  Any advice on how to proceed ?

**Comment by bpbond** [comment]
@pralitp I don't think so. Make your changes, and we can deal with conflicts as necessary. Thanks. We can always re-run the normalizer routine here.

**Comment by bpbond** [comment]
@pralitp Robert's been busy, not a problem but if by any chance you have time to review this I'll get it merged.

**Comment by pralitp** [comment]
Sure, I'll take a look.

File: R/constants.R [hunk 1]
```diff
@@ -353,7 +353,7 @@ CONV_TWH_EJ <- 3.6e-3
 CONV_KWH_GJ <- 3.6e-3
 CONV_GJ_EJ <- 1e-9
 CONV_EJ_GJ <- 1 / CONV_GJ_EJ
-CONV_BBLD_EJYR <- 6.119 * 365.25 * 1e-3 # billion barrels a day to EJ per year
```
Comment (pralitp) [inline R/constants.R]: Was this change..

File: R/zchunk_LA111.rsrc_fos_Prod.R [hunk 2]
```diff
@@ -89,7 +89,7 @@ module_energy_LA111.rsrc_fos_Prod <- function(command, ...) {
       left_join_error_no_match(select(iso_GCAM_regID, iso, GCAM_region_ID), by = "iso") %>%
       # summarise by GCAM region and convert to EJ/yr
       group_by(GCAM_region_ID, year) %>%
-      summarise(value = sum(value) * CONV_BBLD_EJYR) %>%
+      summarise(value = sum(value) * CONV_MBLD_EJYR) %>%
```
Comment (pralitp) [inline R/zchunk_LA111.rsrc_fos_Prod.R]: ..and this change meant for some other PR?

Comment (bpbond) [reply R/zchunk_LA111.rsrc_fos_Prod.R]: This is #864 - I was lazy and tossed them in here.

**Pull request merged**